src/components/TodoList/TodoList.tsx

```diff
@@ -16,35 +23,69 @@
     </thead>
 
     <tbody>
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">1</td>
+      {todos.map(todo => (
+        <tr
+          key={todo.id}
+          data-cy="todo"
+          className={isModal === todo.id ? 'has-background-info-light' : ''}
+        >
+          <td className="is-vcentered">{todo.id}</td>
+          <td className="is-vcentered">
+            {todo.completed && (
+              <span className="icon" data-cy="iconCompleted">
+                <i className="fas fa-check" />
+              </span>
+            )}
+          </td>
+          <td className="is-vcentered is-expanded">
+            <p className={`has-text-${!todo.completed ? 'danger' : 'success'}`}>
+              {todo.title}
+            </p>
+          </td>
+          <td className="has-text-right is-vcentered">
+            <button
+              onClick={() => changeModal(todo.id)}
+              data-cy="selectButton"
+              className="button"
+              type="button"
+            >
+              <span className="icon">
+                <i
+                  className={`far ${isModal === todo.id ? 'fa-eye-slash' : 'fa-eye'}`}
+                />
+              </span>
+            </button>
+          </td>
+        </tr>
+      ))}
+      {/* <tr data-cy="todo" className="has-background-info-light">
+        <td className="is-vcentered">2</td>
         <td className="is-vcentered" />
         <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">delectus aut autem</p>
+          <p className="has-text-danger">quis ut nam facilis et officia qui</p>
         </td>
         <td className="has-text-right is-vcentered">
           <button data-cy="selectButton" className="button" type="button">
             <span className="icon">
-              <i className="far fa-eye" />
+              <i className="far fa-eye-slash" />
             </span>
           </button>
         </td>
-      </tr>
-      <tr data-cy="todo" className="has-background-info-light">
-        <td className="is-vcentered">2</td>
+      </tr> */}
```

Comment:
There is a block of commented-out code here. If this code is not needed, consider removing it to keep the codebase clean. If it is needed for future reference, add a comment explaining why it is commented out.